addons/mail/static/src/components/chatter_topbar/chatter_topbar_tests.js

```diff
@@ -160,7 +145,7 @@ QUnit.test('attachment loading is delayed', async function (assert) {
         loadingBaseDelayDuration: 100,
         async mockRPC(route) {
             if (route.includes('ir.attachment/search_read')) {
-                return new Promise(() => {}); // simulate long loading
+                await makeTestPromise(); // simulate long loading
```

Comment:
not a big deal, but I prefer former LOC because was as short at this while being transparent in what it does. Here, we don't need `makeTestPromise()` for it's intent (public `.resolve()`/`.reject()` methods), and it adds one layer of indirection, with barely any text code character gained...

Reply:
This is related to the other tests that do make use of it.

I saw this one as a particular case of `makeTestPromise` where we just don't ever call the `resolve` method, while some other tests do call it.

(and we don't have tests where `reject` is called, and the business code is not even handling it, but that's another issue)